app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,15 @@
+class LikesController < ApplicationController
+  def create
+    @user = User.find(params[:user_id])
+    @post = @user.posts.find(params[:post_id])
+    @like = @post.likes.build(user: current_user)
+
+    respond_to do |format|
+      if @like.save
+        format.js { render js: 'window.location.reload();' } # Reload the page with JavaScript and update likes
+      else
+        format.js # Render JavaScript to handle the failure scenario if needed
+      end
+    end
```

Comment:
- [ ] [REQUIRED] You have done great work so far @Ayokunnumi1 , however, since this milestone is supposed to test your ROR skills, let's refactor this part where you have used Javascript to use ROR to redirect after the like has been saved or not saved.
Example:
``` ruby
respond_to do |format|
        if @like.save
          format.html { redirect_to @post, notice: 'Like was successfully created.'}
        else
          format.html { redirect_to @post, alert: 'Failed to create like.' }
        end
      end
```